src/opnsense/mvc/app/controllers/OPNsense/Base/ApiMutableTableModelControllerBase.php

```diff
@@ -91,28 +91,35 @@ protected function setItemActionHook($mdl)
      */
     public function setItemAction($uuid)
     {
+        $result = array("result"=>"failed");
         if ($this->request->isPost() && $this->request->hasPost(static::$internalModelName)) {
-            $mdl = $this->getModel();
             if ($uuid != null) {
                 $node = $this->getNodeByUUID($uuid);
                 if ($node != null) {
                     $node->setNodes($this->request->getPost(static::$internalModelName));
-                    return $this->save($mdl, $node, $this->setItemActionHook);
```

Comment:
This func call was with the wrong signature. Also, validate was missing, and needed to add other hooking method.